utils/mil_msgs/msg/LabeledImage.msg

```diff
@@ -0,0 +1,7 @@
+# Represents the complete data from a labeled image from LabelMe
+
+sensor_msgs/Image image            # Image where objects are labeled
```

Comment:
Would this be nessicary? What about a timestamp and topic to get the image from? This would greatly increase the size of bags.